README.md

```diff
@@ -75,34 +75,143 @@ However, there might be cases in which not-yet released demos or features are on
-  ```
-  rviz2 --display-config `ros2 pkg prefix rrbot_description`/share/rrbot_description/config/rrbot.rviz
-  ```
+## *RRBot*
 
 *RRBot*, or ''Revolute-Revolute Manipulator Robot'', is a simple 3-linkage, 2-joint arm that we will use to demonstrate various features.
 It is essentially a double inverted pendulum and demonstrates some fun control concepts within a simulator and was originally introduced for Gazebo tutorials.
 The *RRBot* URDF files can be found in the `urdf` folder of `rrbot_description` package.
 
+1. To check that *RRBot* descriptions are working properly use following launch commands:
+
+   *RRBot*
+   ```
+   ros2 launch rrbot_description view_robot.launch.py
+   ```
+   **NOTE**: Getting the following output in terminal is OK: `Warning: Invalid frame ID "odom" passed to canTransform argument target_frame - frame does not exist`.
+             This happens because `joint_state_publisher_gui` node need some time to start.
+
+1. To start *RRBot* example open open a terminal, source your ROS2-workspace and execute its launch file with:
+   ```
+   ros2 launch ros2_control_demo_bringup rrbot.launch.py
+   ```
+   The launch file loads and starts the robot hardware, controllers and opens `RViz`.
+   In starting terminal you will see a lot of output from the hardware implementation showing its internal states.
+   This is only of exemplary purpuses and should be avoided as much as possible in a hardware interface implementation.
+
+   If you can see two orange and one yellow rectangle in in `RViz` everything has started properly.
+   Still, to be sure, let's introspect the control system before moving *RRBot*.
+
+1. Check if the hardware interface loaded properly, by opening another terminal and executing:
+   ```
+   ros2 control list_hardware_interfaces
+   ```
+   You should get:
+   ```
+   command interfaces
+        joint1/position [claimed]
+        joint2/position [claimed]
+   state interfaces
+         joint1/position
+         joint2/position
+
+   ```
+   Marker `[claimed]` by command interfaces means that a controller has access to command *RRBot*.
+
+1. Check is controllers are running:
+   ```
+   ros2 control list_controllers
+   ```
+   You should get:
+   ```
+   joint_state_broadcaster[joint_state_broadcaster/JointStateBroadcaster] active
+   forward_position_controller[forward_command_controller/ForwardCommandController] active
+   ```
+
+1. If you get output from above you can send commands to *Forward Command Controller*, either:
+
+   a. Manually using ros2 cli interface:
+   ```
+   ros2 topic pub /forward_position_controller/commands std_msgs/msg/Float64MultiArray "data:
+   - 0.5
+   - 0.5"
+   ```
+   B. Or you can start a demo node which sends two goals every 5 seconds in a loop:
+   ```
+   ros2 launch ros2_control_demo_bringup test_forward_position_controller.launch.py
+   ```
+   You should now see an orange box circling in `RViz`.
```

Comment:
I think that this sentence is incorrect for the rrbot robot.

Reply:
```suggestion
   You should now see orange and yellow blocks moving in `RViz`.
```